pkg/api/service/util.go

```diff
@@ -18,17 +18,18 @@ package service
 
 import (
 	"fmt"
-	api "k8s.io/kubernetes/pkg/apis/core"
-	netsets "k8s.io/kubernetes/pkg/util/net/sets"
 	"strings"
+
+	api "k8s.io/kubernetes/pkg/apis/core"
+	netsets "k8s.io/utils/net"
```

Comment:
would change this to 
```
netutils "k8s.io/utils/net"
```
or just 
```
"k8s.io/utils/net"
```
(though this will often be confused with `net` in stanrd library). 

That way we're not referencing "set" twice when calling i.e. netsets.IPNetSet

Reply:
yeah nice idea

Reply:
but @andrewsykim we cannot do this in all places as it conflicts with the standard net package. should we use a better name?

Reply:
`netutils "k8s.io/utils/net"` should be fine I think.

Reply:
when it conflicts use `netutils "k8s.io/utils/net"`

Reply:
i used utilnet